website/client/components/chat/autoComplete.vue

```diff
@@ -25,9 +25,15 @@ export default {
   },
   computed: {
     autocompleteStyle () {
+      function heightToUse (topCoords) {
+        let textAreaHeight = document.getElementsByClassName('user-entry')[0].clientHeight;
```

Comment:
I think `document.getElementsByClassName('user-entry')` could be passed as a prop (using `$refs` instead of going through the DOM)